Classifier and Summarizer/WebEntityDetection.py

```diff
@@ -0,0 +1,128 @@
+                {
+                    "maxResults": self.max_entities,
+                    "type": "WEB_DETECTION"
+                }
+            ],
+            "imageContext": {
+                "webDetectionParams": {
+                    "includeGeoResults": "true"
+                }
+            }
+        }
+
+    def prepare_requests(self, encoded_images: list) -> list:
+        '''
+        prepares the requests for making calls
+        to the api. Structures the given images
+        in the format as expected by the API
+        Params :
+          encodedImages : list of base64 encoded images
+        Return Type:
+        list: of required format
+        '''
+        image_requests = []
+
+        for image in encoded_images:
+            image_requests.append(self.format_single_request(image))
+
+        return image_requests
+    
+    ###   json_response["responses"][image_index]
+
+    def get_best_guess_label(self, json_response_for_image):
+        ''' return the best guess label '''
+        return json_response_for_image["webDetection"]["bestGuessLabels"][0]["label"]
+
+    def get_top_entities(self, json_response_for_image, number_of_entities):
+        ''' return the top entity description'''
+        top_entites = []
+        number_of_entities = min(number_of_entities, len(
+           json_response_for_image["webDetection"]["webEntities"]))
+        
+        for i in range(number_of_entities):
+            if "description" in json_response_for_image["webDetection"]["webEntities"][i]:
+                top_entites .append(json_response_for_image["webDetection"]["webEntities"][i]["description"])
+        return top_entites
+
+    def get_description_for_images(self, images: list) -> list:
+        '''
+        given a list of images returns the best guess
+        labels for the web entities or description of
+        the images
+        Params:
+          images : list of image paths
+        Return type:
+        list<str> : list containing best guess descriptions of the image
+        '''
+        base64_encoded_images = self.find_base64_encoding_for_images(images)
+
+        image_requests = self.prepare_requests(base64_encoded_images)
+
+        json_data_for_post_request = json.dumps({
+            "requests": image_requests
+        })
+        response = requests.post(self.api_url, data=json_data_for_post_request)
+        
+        if response.status_code != 200:
+            return "BadRequestError"
+
+        response = json.loads(response.content)
+        image_descriptions = []
+        for i in range(len(images)):
+            image_descriptions.append(
+                {
+                    "label": self.get_best_guess_label(response["responses"][i])  ,
+                    "entities": self.get_top_entities(response["responses"][i], self.max_entities ) # pass on only response for the image
+                    # add more fields / attributes as required
+                }
+            )
+
```

Comment:
@tmadhuri - Changed the data returned in a dict format. This would make it flexible to add an other field if required